src/testdir/test_codestyle.vim

```diff
@@ -166,14 +166,17 @@ enddef
 def Test_indent_of_source_files()
   for fname in glob('../*.[ch]', 0, 1) + ['../xxd/xxd.c']
     execute 'tabnew ' .. fname
+    if &expandtab
+      continue
+    endif
     for lnum in range(1, line('$'))
       var name: string = synIDattr(synID(lnum, 1, 0), 'name')
       if -1 == index(['cComment', 'cCommentStart'], name)
         var line: string = getline(lnum)
         var indent: string = matchstr(line, '^\s*')
         var tailing: string = matchstr(line, '\s*$')
         if !empty(indent)
-          if indent !~# '^\t* \{0,7\}$'
+          if indent !~# '^\t* \{0,7}$'
```

Comment:
The result is the same, but the correct notation for the preceding atom is `\{}`.
See `:h /\{`